spec/brcobranca/boleto/sicoob_spec.rb

```diff
@@ -74,10 +74,8 @@
     expect(boleto_novo.agencia_conta_boleto).to eql("3007 / 0318957")
   end
 
-  it "Busca logotipo do banco" do
-    boleto_novo = described_class.new
-    expect(File.exist?(boleto_novo.logotipo)).to be_truthy
-    expect(File.stat(boleto_novo.logotipo).zero?).to be_falsey
+  describe 'Busca logotipo do banco' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.